pom.xml

```diff
@@ -47,10 +47,10 @@
 			xAzureSQLDB - - - - For tests not compatible with Azure SQL Database - - 
 			xAzureSQLDW - - - - For tests not compatible with Azure Data Warehouse - 
 			xAzureSQLMI - - - - For tests not compatible with Azure SQL Managed Instance 
+			NTLM - - - - - - - For not running NTLM tests (excluded by default) - - - 
```

Comment:
`xNTLM`?

Reply:
x Notation is for "Not compatible with XXX" tests..
NTLM is fine. Since the tag is for tests that do NTLM Auth.

Reply:
Description should be updated to something like "To enable NTLM tests".

Reply:
Yes.. Description should be fixed to: "For tests using NTLM Authentication mode"
Enabled/Not Enabled (verb) depends on if one puts this tag in "excludedGroups".